generic-styles/index.less

```diff
@@ -79,14 +79,18 @@ body {
   }
 
   span {
+    &.os-icons {
+      display: flex;
+      float: left;
+    }
```

Comment:
These styles are no longer applied, because `.os-icons` wrapper is a `div`.

Reply:
That's right - CE is going to update the structure so it will work once we have a new pipeline.

Reply:
New structure ([per this PR](https://github.com/openstax/recipes/pull/165/files)):
```
<p class="annotation dreaming-icon" id="auto_88eb0fc4-6144-4c92-84ff-56f8f811e4f7_para-00005">
   <span class="os-icons">
      <span aria-label="Generating and capturing ideas icon" class="dreaming-icon" role="img" title="Generating and capturing ideas icon"></span>
   </span>
   <span class="os-text">In other words, the more you write, the more confidence you will have in your voice and your writing.</span>
</p>
```